### PR TITLE
Typos fixes

### PR DESCRIPTION
**Pull request by Zeitsperre**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,24 @@
 input_data
 public/data
+public/indicateurs
```
Comment (grassick) [inline .gitignore]: These folders shouldn't exist, so don't need to be here, I think

File: .gitignore [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+public/statiques
 
 /node_modules
 
 # production
 /build
 
+# geometry files
+*.geojson
```
Comment (grassick) [inline .gitignore]: I don't think we should ignore wildcards like this.

File: .idea/.gitignore [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (grassick) [inline .idea/.gitignore]: This entire file can do, as .idea is already ignored above

**Review by grassick — CHANGES_REQUESTED** [review]
Can you clear out the .gitignore changes? If it's tricky, I can merge and then revert them myself.

**Comment by Zeitsperre** [comment]
I realize now that I could've reverted those commits... If you perform a squash, the history will be kept clean. Thanks again!

**Pull request merged**